pkg/libinit/root_linux.go

```diff
@@ -284,3 +295,71 @@ func InstallModules(pattern string, exclude map[string]bool) error {
 
 	return nil
 }
+
+func readModules(f *os.File) []string {
+	r := bufio.NewReader(f)
+	modules := []string{}
+	for {
+		if i, err := r.ReadString('\n'); err == nil {
```

Comment:
Does it make sense to use strings.TrimSpace to remove leading and trailing whitespace?